src/auth/auth.ts

```diff
@@ -57,11 +62,11 @@ export class BBAuth {
     }
 
     if (match) {
-      if (match[3]) {
-        zone = match[3].substring(1);
+      if (match[ZONE_INDEX]) {
+        zone = match[ZONE_INDEX].substring(1);
```

Comment:
nitpick: this was already here, but could we restructure the regex such that the `-` in the hyphen doesn't get captured, thus negating the need for the substring?

Reply:
There is a reason I couldn't do that.  But I'll have to look again.

Reply:
updated